### ISSUE TITLE
Allow multiple ui.frontend modules

### ISSUE THREAD

**Issue by brunob15** [adobe/aem-project-archetype#940]
### Expected Behaviour
The _frontendModule_ parameter should allow more than one value (for example, _general_ and _angular_), so we can create a project that supports both SPA Editor and traditional AEM.

### Actual Behaviour
The _frontendModule_ parameter allows one single value, so currently is not possible to create a project that combines SPA and non-spa.

**Comment by brunob15** [comment]
Any thoughts? @bpauli @gabrielwalt @vladbailescu 
If this is considered an enhancement, this is something I'd like to work on.

Thanks!

**Comment by kevinolivar** [comment]
@brunob15 mhhh can't you just create another module without using the frontendModule option? Eg. copy paste from either general or react

**Comment by brunob15** [comment]
@kevinolivar yes, afaik that's the way one should follow if we want to do it currently. But the ui.frontent module is not the only thing that changes when you create a SPA project. If you do a search in the source code, there are several things to copy/merge:

![image](https://user-images.githubusercontent.com/5839627/174351931-ba9624d5-0a55-43f7-83e3-a6b6b62d3880.png)

That's why I think it would be nice to have the option to do it automatically.

**Comment by kevinolivar** [comment]
I guess this should be tagged as 'enhancement'.

**Comment by mitchross** [comment]
I need this also, does anyone have a solution ?
